code/game/objects/items/weapons/weaponry.dm

```diff
@@ -221,7 +221,7 @@
 
 /obj/item/weapon/cane
 	name = "cane"
-	desc = "A cane used by a true gentlemen. Or a clown."
+	desc = "A cane used by a true gentleman. Or a clown."

```

Comment:
I really don't think it matters if you put a one character spelling fix in your PR but leave it out of the OP